survey/locales/en/customServer.yaml

```
@@ -49,3 +49,9 @@ resetPasswordEmailText: >
 
 
   Thank you
+confirmedByAdminEmailSubject: "National survey admin platform: Account confirmed"
+confirmedByAdminEmailText: >
+  Hello {{name}},
+
+
+  Your access request to the survey platform has been confirmed. You can now use your account.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)